### PR TITLE
Feature final pbi updates

### PR DESCRIPTION
**Pull request by mnallajerla**

### CODE REVIEW

File: red/part2/part2_learner/mtl_red_part_2_see.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ output:
      - In the event they do not, to make clinic selections yourself, you can select the same set of clinics you chose in Part 1 of _Modeling to Learn_.
```
Comment (lijenn) [inline red/part2/part2_learner/mtl_red_part_2_see.md:57]: Hi @mnallajerla - I couldn't make a comment in some of the lines , so I'm making it all in line 57 for now:
- [ ] Line 54: Change "At mtl.how/data_test..." to "At mtl.how/data..."
- [ ] Line 44: Language is implying that we're making Clinic Selections again, would rephrase it "
- [ ] General: When not referring to Clinic Selection as the tab, I would include the space (e.g. How we write Clinic Selections in Line 42 and 44 is different than line 56. But in Line 55, I would keep the space since it's referring to the ClinicSelection tab).
- [ ] Continue working on editing 2.0 or 2.5 to 3.0

Lmk if you have any questions, thanks!
_________________________________________________________________________
For @lijenn to do BEFORE merging pull request:
- [ ] add data ui cheatsheet to red part ii and blue s3 folders
- [ ] add cpt code cheatsheet 
- [ ] facilitator one pager

**Comment by lijenn** [comment]
Hey @mnallajerla! I left some feedback in each of the Red Part II guides (or scroll up in this pull request).
Would you please mind making those edits and looking out for those edits/similar edits as needed in the Blue Session 3 guides too? 

 - [ ] I'll fix the old links/logos in the Red Part I and Blue Session 2. 
- [ ] I'll also ping you when the Landing Page is ready for GIFS.

Let me know if you need any help. THANK YOU :)

**Comment by mnallajerla** [comment]
@lijenn all done :) 

I finished all your edits. Did you have any edits for Red Part 1 or blue S2? Otherwise, we are ALL SET! :)

**Comment by lijenn** [comment]
@lijenn Final edits before merging Red Part II and Blue Session 3 to QA then Master:
- [x] References to the Data UI (change to Viz and Data tabs) and Team Data Table (change to Team Data)
- [x] Timestamps
- [x] Remove Sim UI references from Red Part II
- [x] Edit all references to 2.5 to 3.0
- [x] Edit READMEs from 2.0 to 3.0
- [x] AGG -> Team (Agg) Care
- [x] SP -> Team (SP) Flow
- [x] DataNotes -> Data Notes 1 and Data Notes 2
- [x] References to legacy columns -> Descriptions and Details tabs. 
- [x] Done/Do logos for Blue Session 3 should have MTL Sim and MTL Menu
